package.json

```diff
@@ -115,7 +115,7 @@
     "strip-json-comments": "^3.1.1",
     "terser-webpack-plugin": "^3.0.5",
     "typedoc": "^0.19.2",
-    "typescript": "^4.0.5",
+    "typescript": "4.0.5",
```

Comment:
I set this to `4.0.x` in js-ipfs when I encountered similar problems.

Reply:
ah, because our awesome smart linter doesnt allow ~ right ill do that